src/elements/atoms/Link/Link.css

```diff
@@ -18,4 +18,9 @@
       color: var(--color-black);
     }
   }
+
+  &--icon {
+    display: flex;
+    width: fit-content;
```

Comment:
- I like the thought here, but this is not supported in IE11 unfortunately and will have to be fixed per project. https://caniuse.com/#search=fit-content

- Let use flex class vs. styles. I updated the flex modifier to be by default on across all breakpoints, and now have flex--{size} classes to use if you need it only for a particular breakpoint (aka better now than our first attempt on Aptive).